resources/instance_analytics/collections/vG58R8k-QddHWA7_47umn_metabase_analytics/dashboards/BHyad8ZHCbeiBZpQxDwsP_content_with_cobwebs.yaml

```diff
@@ -28,64 +27,5 @@ serdes/meta:
 - id: BHyad8ZHCbeiBZpQxDwsP
   label: content_with_cobwebs
   model: Dashboard
-dashcards:
```

Comment:
not sure what is going on here cc @piranha @crisptrutski

Reply:
uhhh this is very concerning; are you sure it's 50.22? I've just checked locally and I get those dashcards...